### PR TITLE
deDE update

### PR DESCRIPTION
**Pull request by Dynaletik**

### CODE REVIEW

**Comment by Dynaletik** [comment]
Sorry, was on vacation for some weeks. :D

File: Aberrus/Locales/deDE.lua [hunk 1]
```diff
@@ -1,106 +1,106 @@
...
 end
 
 L = BigWigs:NewBossLocale("Assault of the Zaqali", "deDE")
 if L then
 	-- These are in-game emotes and need to match the text shown in-game
 	-- You should also replace the comment (--) with the full emote as it shows in-game
-	--L.zaqali_aide_north_emote_trigger = "northern battlement" -- Commanders ascend the northern battlement!
-	--L.zaqali_aide_south_emote_trigger = "southern battlement" -- Commanders ascend the southern battlement!
+	L.zaqali_aide_north_emote_trigger = "nördliche Festungsmauer" -- Commanders ascend the northern battlement!
+	L.zaqali_aide_south_emote_trigger = "südliche Festungsmauer" -- Commanders ascend the southern battlement!
```
Comment (funkydude) [inline Aberrus/Locales/deDE.lua:42]: We want the full ingame emote in the comment here

Comment (Dynaletik) [reply Aberrus/Locales/deDE.lua]: Done ;)

Comment (funkydude) [reply Aberrus/Locales/deDE.lua]: No, in the comments, not in the translation itself

Comment (Dynaletik) [reply Aberrus/Locales/deDE.lua]: Okay.

**Comment by funkydude** [comment]
Excellent work! Thank you!

**Pull request merged**